nujo/utils/data/data_loader.py

```diff
@@ -0,0 +1,25 @@
+from os import mkdir
+from os.path import exists, expanduser
+import requests
+
+
+class DataLoader:
+    def __init__(self, name):
+        self._name = name
+        self._link = 'https://archive.ics.uci.edu/ml/machine-learning-databases/{0}/{0}.data'.format(
+            name)
```

Comment:
- [ ] @ManiacMaxo Use f-strings, not `.format` (f strings are faster and cooler 😃).  Check out how f-strings work [here](https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/).

- [ ] Move `https://archive.ics.uci.edu/ml/machine-learning-databases/` in a separate file as a constant